### DISCUSSION TITLE
hoppscotch usage problem

### DISCUSSION THREAD

**Discussion by simonli866** [hoppscotch/hoppscotch#2278] [Q&A] [answered]
can't access property "content-type",syntheticResponse. headers is undefined 
![图片](https://user-images.githubusercontent.com/22612182/164172479-37556297-7e2a-46b6-8813-76ef4a8bfab8.png)
![图片](https://user-images.githubusercontent.com/22612182/164172843-aac03e1b-34fb-4574-a505-b4b449074c58.png)
Is it because of some mistake and I have not set up any agents

**Comment by liyasthomas** [comment, marked as answer]
Seems like you're using an older build. Please use the latest production channel on [hoppscotch.io](https://hoppscotch.io).
